docs/scheduling.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Scheduling
+---
+
+Scheduling guidelines
+-------------
+
+### Access-tier based volume scheduling
+
+DirectCSI provides a way to restrict scheduling of volumes to only selective drives based on the 'access-tier' tag set on them.
```

Comment:
In addition to scheduling based on resource constraints (available space) and node topology (affinity/anti-affinity etc.), it is possible to further influence the scheduling of workloads to specific volumes based on "access-tiers". DirectCSI pre-defines 3 access tiers:
 - Hot
 - Warm
 - Cold

By default, direct-csi drives are not associated with any access-tier. An admin can choose drives to associate to a particular access tier. Further instructions on how is provided in the following sections